### DISCUSSION TITLE
Not able to download data from MINIO

### DISCUSSION THREAD

**Discussion by balajisankaran87** [minio/minio#19962] [Q&A]
Hi Team,

Backup is performed on IBM i system using ICC and BRMS. On IBM i partition, we have doubled the system storage. 
ICC requires a S3 storage configuration for which we have installed RHEL server hosting MINIO software. MINIO converts RHEL storage as S3 storage. 
Once the backup runs on IBM i partition, it saves the data on virtual tapes which is stored within the IBM i partition. 
Then ICC will transfer these virtual tapes to MINIO (which is installed on RHEL server). 
For restoration we need to download the required virtual tapes from RHEL server to IBM i partition.

Currently we can successfully transfer the backup tapes (any size) to MINO object storage from IBM i however we are not able to download the backup tapes from MINIO back to IBM i partition. These transfers from MINIO to IBM i partition behavior is erratic.
 
We have the default MINIO configuration and not sure what needs to be modified here. 
Could you please help us with the MINIO configuration? so that we can download the data without any issues.

**Comment by klauspost** [comment]
I can't judge how supported/unsupported your case is, and I suspect it will require a deeper investigation.

It seems reasonable to take this as a more commercial request. Please reach out on `sales@min.io` and we can see what we can do to assist you in this setup.
